code.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "name": "regional-hydrologic-forcings-ml",
+    "organization": "U.S. Geological Survey",
+    "description": "Repo for machine learning models for regional prediction of hydrologic forcing functions. FY22 regions: Delaware River Basin (DRB) region, and Upper Colorado River Basin (UCOL) region.",
+    "version": "main",
+    "status": "Development",
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://github.com/USGS-R/regional-hydrologic-forcings-ml/blob/main/LICENSE.md"
+        }
+      ]
+    },
+    "homepageURL": "https://github.com/USGS-R/regional-hydrologic-forcings-ml/",
+    "downloadURL": "https://github.com/USGS-R/regional-hydrologic-forcings-ml/archive/refs/heads/main.zip",
+    "disclaimerURL": "https://github.com/USGS-R/regional-hydrologic-forcings-ml/blob/main/DISCLAIMER.md",
+    "repositoryURL": "https://github.com/USGS-R/regional-hydrologic-forcings-ml.git",
+    "vcs": "git",
+    "laborHours": -1,
+    "tags": [
+      "R",
+      "USGS",
+      "streamflow metrics",
+      "random forest",
+      "hierarchical clustering"
+    ],
+    "languages": [
+      "R"
+    ],
+    "contact": {
+      "name": "Jared D. Smith",
+      "email": "jsmith@usgs.gov"
+    },
+    "date": {
+      "metadataLastUpdated": "2023-04-07"
+    }
+  },
+  {
```

Comment:
@jesse-ross what is the second element in this list? The first element is our main repo. I do not see a `v0.0.0` tag in our repo

Reply:
It is intended for other branches or releases. It's populated with v0.0.0 just as a placeholder to show the right structure if you want to have multiple chunks. However, in many data science projects, `main` is the only important branch, and there isn't formalized set of releases. If your project is like that, you could just remove the second element.